axiom/nr_app_harvest.c

```diff
@@ -12,6 +12,21 @@
 #include "nr_app_harvest_private.h"
 #include "util_logging.h"
 
+/*
+ * The build pipeline for PHP 8.2 uses a Debian image that is built on a
+ * version of glibc that contains an newer glibc version symbol for `pow()`
+ * which is incompatible with older distributions, causing RPM installs to fail.
+ *
+ * To pin the version of `pow` to a version compatible with all NR PHP Agent
+ * supported OSes, we utilize `asm()` and `.symver` to instruct the linker to
+ * select an older version- in this case, 2.17.
+ */
+#if defined(__GLIBC__)
```

Comment:
```suggestion
#ifdef __GLIBC__
#if __GLIBC__ >= 2
#if __GLIBC_MINOR__ > 17
  __asm__(".symver pow,pow@GLIBC_2.17");
#endif
#endif
#endif
```

Reply:
You'll also need to add a 

#if defined(__GLIBC__)
#include <gnu/libc-version.h>
#endif

at the top of the file to get the include with the __GLIBC__ and __GLIBC_MINOR__ defines.

Reply:
First suggestion added in 661beed2b6ba197adefdb386b3a59cbad149a5fe.